templates/outer-loop/Dockerfile

```diff
@@ -42,19 +43,24 @@ COPY --from=compile --chown=1001:0 /work/config/ /config/
 
 # 2d) Add the microprofile health feature configuration if it is not already defined in the user's configuration.
 #     This allows k8s to use the deployment's health probes.
+#     To skip this step, use the ADD_MP_HEALTH argument when building the outer loop image. 
+#     The allowed values are true/false.  The default value is true.
+#     Example usage: docker build --build-arg ARG ADD_MP_HEALTH=false ...
 RUN mkdir -p /tmp/stack/config/configDropins/defaults
 COPY --from=compile --chown=1001:0 /stack/ol/config/configDropins/defaults/ /tmp/stack/config/configDropins/defaults/
 
 ARG ADD_MP_HEALTH=true
 RUN if [ "$ADD_MP_HEALTH" = "true" ]; then \
-        /opt/ol/wlp/bin/server start; \
-        /opt/ol/wlp/bin/server stop; \
+        /opt/ol/wlp/bin/server start && \
+        /opt/ol/wlp/bin/server stop && \
         if ! grep "CWWKF0012I" /logs/messages.log | grep -q 'mpHealth-[0-9]*.[0-9]*\|microProfile-[0-9]*.[0-9]*'; then \
-            echo "Missing mpHealth feature - adding config snippet"; \
+            echo "Missing mpHealth feature - adding config snippet" && \
             cp /tmp/stack/config/configDropins/defaults/liberty-stack-mpHealth.xml /config/configDropins/overrides; \
         else \
             echo "Found mpHealth feature - not adding config snippet"; \
-        fi; \
+        fi && \
+        rm -rf /liberty/output/.classCache && \
```

Comment:
Just curious..was this causing problems or just general cleanup, making the image smaller, etc.?

Reply:
Without that update you get these error messages:

JVMSHRC226E Error opening shared class cache file 
JVMSHRC336E Port layer error code = -102 JVMSHRC337E 
Platform error message: Permission denied 
JVMSHRC840E Failed to start up the shared cache. 
JVMSHRC686I Failed to startup shared class cache. Continue without using it as -Xshareclasses:nonfatal is specified